src/sentry/api/endpoints/sentry_app_details.py

```diff
@@ -14,11 +14,11 @@
 
 
 class SentryAppDetailsEndpoint(SentryAppBaseEndpoint):
-    def get(self, request, sentry_app):
-        return Response(serialize(sentry_app, request.user, access=request.access))
+    def get(self, request, sentry_app, access):
+        return Response(serialize(sentry_app, request.user, access=access))
```

Comment:
Was this change necessary? I think in `sentryapps.py` the access kwarg is being set to request.access anyway right?